azure-pipelines.yml

```diff
@@ -117,6 +117,10 @@ stages:
           image: tfcollins/libiio_ubuntu_18_04-ci-arm-ppc:latest
           arch: aarch64
           artifactName: 'Ubuntu-arm64v8'
+        bullseye:
```

Comment:
Can you change name to have the same format: "debian_bullseye_arm64v8"?
And I think you can add a build for 32bit image  just by specifying "arch: arm"